packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -668,6 +668,19 @@ describe('RPC Server Acceptance Tests', function () {
         });
     });
 
+    describe('eth_feeHistory', () => {
+        it('should call eth_feeHistory', async function() {
+            const res = await relay.call('eth_feeHistory', []);
+
+            expect(res.baseFeePerGasArray).to.exist.to.be.an('Array');
+            expect(res.gasUsedRatioArray).to.exist.to.be.an('Array');
+            expect(res.oldestBlockNumber).to.exist;
+            expect(res.baseFeePerGasArray[0]).to.equal('0xa7a3582000');
```

Comment:
- What is this hardcoded value? I think that it is generally better to represent the value in decimal and hex it using `ethers.util.hexValue`
- Is this value something that is hardcoded / used within the relay? if it is, lets use the value from the relay (maybe config property?) because if we change the value in the relay, the tests will start to fail